pkg/features/antrea_features.go

```diff
@@ -54,7 +54,7 @@ const (
 	NetworkPolicyStats featuregate.Feature = "NetworkPolicyStats"
 
 	// alpha: v0.13
-	// Expose Pod IPs through NodPort
+	// Expose Pod Ports through NodPort
```

Comment:
Is "Ports" Capitalized on purpose?

Reply:
@tnqn had suggested this comment, though I'm not sure if we can change from capital to lower case.

Reply:
Sorry, I meant to change "IPs" to "ports", the capital was a mistake. And another typo here: "NodPort".

Reply:
Thanks. Updated.